core/trino-main/src/test/java/io/trino/sql/query/TestJsonQueryFunction.java

```diff
@@ -425,7 +425,7 @@ public void testNullInput()
     }
 
     @Test
-    public void testDescendantMemberAccessor()
+    public void testDescendantMemberWithArrayWrapperAccessor()
```

Comment:
Rename to `testDescendantMemberAccessorWithArrayWrapper()`. The `..` operator in an accessor.